src/sinks/influxdb/logs.rs

```diff
@@ -235,6 +240,31 @@ mod tests {
         toml::from_str::<InfluxDBLogsConfig>(&config).unwrap();
     }
 
+    #[test]
+    fn test_encode_event_apply_rules() {
+        let mut event = Event::from("hello");
+        event.as_mut_log().insert("host", "aws.cloud.eur");
+        event.as_mut_log().insert("timestamp", ts());
+
+        let mut sink = create_sink(
+            "http://localhost:9999",
+            "my-token",
+            ProtocolVersion::V1,
+            "ns",
+            vec![],
+        );
+        sink.encoding.except_fields = Some(vec![Atom::from("host")]);
```

Comment:
Are we asserting that `host` is not included in the output?

Reply:
Yes, we asserting that `host` is not included